scripts/ccpp_datafile.py

```diff
@@ -346,6 +346,11 @@ def datatable_report(datatable, action, sep):
         emsg += _command_line_parser().format_usage()
         raise ValueError(emsg)
     # end if
+    if not sep:
+        emsg = "datatable_report: An separator character (<sep>) is required\n"
```

Comment:
I think this should be "A separator" and not "An separator"

Reply:
Oops, that is a error.